package.json

```diff
@@ -27,6 +27,8 @@
     "test": "yarn workspace snap run test"
   },
   "devDependencies": {
+    "@lavamoat/allow-scripts": "^3.0.0",
```

Comment:
Let's remove allow-scripts from the snap package then as it should be applied at the root